docs/governance/README.md

```diff
@@ -0,0 +1,116 @@
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **Acceptance requires**
+  - A **sponsorship from a board member**. That person will be assigned the
+    issue and is expected to shepard the issue until it's easier accepted or
+    rejected.
+  - A **prototypic implementation** in at least one .NET implementation. The
+    implementation must be open source so that other .NET implementations can
+    easily jump-start their own implementations or take it as-is.
+* **.NET Standard updates are planned** and will generally follow a set of
+  themes. We avoid grab-bag releases of .NET Standard and instead try to define
+  a set of goals that describe what kind of feature areas a particular .NET
+  Standard version provides. This simplifies answering the question which .NET
+  Standard a given library should depend on. It also makes it easier for .NET
+  implementations to decide whether it's worth implementing a higher version of
+  .NET Standard.
+* **The version number** is subject to discussion and is generally a function of
+  how significant the new version is. While we aren't planning on making
+  breaking changes, we'll rev the major version if the new version adds large
+  chunks of APIs or has sizable changes in the overall developer experience.
+
+[netstandard-api]: https://github.com/dotnet/standard/issues?q=is%3Aopen+is%3Aissue+label%3Anetstandard-api
+
+## Inclusion principles
+
+Not every .NET API needs (or even should be) part of the .NET Standard.
+
+If a library is written in pure IL (for example C#, VB.NET, F# etc.), and
+targets .NET Standard, then it can run across all current and future .NET
+implementations with no code changes. The only requirement on the .NET
+implementation is that it has to support at least the version of .NET Standard
+that the library was compiled for (or a higher version).
+
+Any modification to such a library, whether it's bug fixes, performance
+improvements, or additional APIs, will also work across all these .NET
+implementations as well. That means the library author has flexibility to make
+releases on their own cadence while consumers of the library have the choice on
+when they upgrade the version they are using. If the library is made a part of
+.NET Standard, it can only version at the same cadence as the standard, so the
+update flexibility is much reduced.
+
+On the other hand, there are also downsides for APIs not being part of the
+standard. If an API ships as a library, it's much harder for any .NET
+implementation to use the types as part of their implementation or public API
+surface. Hence, there is a lot of value in adding widely used concepts to the
+.NET Standard in order to use them throughout all .NET implementations.
+
+The following criteria helps us to identify APIs that should be part of .NET
+Standard:
+
+* **Ubiqutous APIs**. APIs in the .NET Standard must be implemented by all .NET
```

Comment:
What constitutes "all" .NET implementations? Windows 8.x, Windows Phone, and Windows Phone Silverlight were dropped. Is it only .NET implementations that have current LTS?

Say Everyone But .NET Framework can implement some new API, does full .NET get left behind?

Reply:
This only covers implementations that are still evolved. If a platform is done, that naturally means it will no longer implement later versions of the .NET Standard spec either, so the standard no longer has to consider them either.